### ISSUE TITLE
"a few", "a little"

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#170]
I would like to discuss the analysis of "a few" (and later "a little").

Typically it modifies a plural noun, in which case the current analysis almost across the board in EWT and GUM is to attach "a" and "few" separately to the head noun, with `det` and `amod` respectively.

I'm not sure this is the best analysis, however: consider that "a few" can stand alone for the NP, similar to "some" or "several":

- There were apples on sale. I bought a few.

"A few" can also be coordinated with quantifiers/quantities:

- Did you buy many or just a few?
- Did you buy one or a few? (cf. GUM example)

It can be followed by an "of"-PP:

- a few of the apples

And it can modify things other than plural nouns:

- [a few more] apples
- [a few hundred] apples

The "a" cannot be omitted or replaced with "the" and receive the same interpretation, nor can a possessive + few receive that interpretation:

- I bought John's few apples ≠ I bought a few of John's apples

Taken together, the expression functions a lot like determinative "many".

"A little" behaves very similarly except for mass rather than plural nouns:

- There was ice cream on sale. I bought a little.
- Did you buy much or just a little?
- A little of the ice cream was leaking out.
- a little more ice cream

**Would it be better to analyze these as `fixed(a, few)`, `fixed(a, little)`?**

Here are the exceptions to the typical analysis in the corpus (including a couple of errors):

- EWT: http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=608ffac86446a&clustering=N2.lemma
- GUM: http://match.grew.fr/?corpus=UD_English-GUM@dev&custom=608ffade31c10&clustering=N2.lemma

Note that there are other quantificational expressions with "a" + NOUN—a couple, a bit, a lot, a bunch. These already form a constituent with the article attaching to the noun as `det`.

Internal modification: some of these expressions allow intensification, but these may be lexicalized expressions as well. The ADJ ones seem less flexible than some of the NOUN ones:

a + ADJ
- a fair few
- ?a very few/little

a + NOUN
- a whole lot
- a whole bunch, ?a huge bunch, ?a great bunch
- a little bit, a tiny bit, etc.

**Comment by nschneid** [comment]
@amir-zeldes any opinions?

**Comment by amir-zeldes** [comment]
Well, I agree that from a historical perspective the current analysis is a bit simplistic, but I think it is mainly motivated by a "no better solution" situation: in many ways, it's like "a lot of" but without the "of", so you can't do a normal `nmod` with "few" as the head. For "a few of the apples" I think the analysis should parallel "a lot of".

The fact that "a few" can stand by itself is true, but that doesn't mean it has to have the same analysis. Semantically it is pretty similar to "many", as you say, or also "some" as a quantity expression.

I think I could see doing "a <-det few" instead of attaching to the noun, but I would be opposed to `fixed` since there is definitely the possibility of modification, here's a random example:

  * Bradfield supplies **a scant few** orienting facts in the course of this challenging, original novel

And in general, I would say it's relatively clear "few" outranks "a" as a possible head, so it's not really 'structureless'. The only real question is 'who does the "a" belong to'.

The main reasons I would hesitate to change this are:

  * It's work, so we'd need a good reason to do it
  * I'm not sure how this would help (it's not like the singular determiner makes more sense for "few" than the main noun)
  * Its sort of elegant that if we view "few" as `amod`, it attaches like an adjective, and then if there's an article that attaches to the noung
  * It would be easier to argue that it belongs to "few" if we could add a second article for the noun, but we can't say "the a few books", or "some a few books", so it appears to occupy the same slot as the article (alternatively you could say "a few" does that, which I think is what you're saying?)

So I guess it's mainly a question of whether that would make the analysis much better, and I'm not super sure about that. I guess it would be `nmod:npmod` to the noun, right? Or what were you thinking for the function of "a few" with respect to the plural noun?

**Comment by nschneid** [comment]
It occurs to me that this is similar to article+number expressions like "a hundred", which also permit plural noun ellipsis: _I bought a hundred (books)._

GUM treats "a hundred" as a constituent (`det(hundred, a)`, which "hundred" attaching as `nummod`). So treating "a few" and "a little" as constituents would be similar to that.

**Comment by nschneid** [comment]
I agree `det` on an adjective feels kind of wrong, but that's what we are currently forced to do if the head noun is omitted. The `fixed` analysis makes the most sense to me—i.e. capturing that "a few" and "a little" are two syntactically very special multiword expressions combining what was historically a determiner and an adjective. ("A lot" and friends are *semantically* special but not as much syntactically because "lot" is historically a noun.)

**Comment by amir-zeldes** [comment]
`det` on an adjective with no noun is the normal way of handling 'promotion' (or more traditionally, nominalization of the adjective), so I don't find it bad. It's the same as "the poor", or "the following". My concern was having something with deprel `amod` having the `det`, which this would introduce in this construction.

I think I feel pretty strongly against `fixed` for a number of reasons:
  * Whatever we think "scant" is, the construction is modifiable, so you'd be introducing potentially quite a few "fixed" expressions to an already long list
  * If fixed is really only used for things where headedness is meaningless, I don't think this is such a case: I'm pretty sure "few" outranks "a"
  * It obscures the fact that "a" is actually fulfilling a fairly normal role here, and erases the parallelism with "few" without an article. Currently we have "few dogs" and "a few dogs" being structurally very similar (both have an adjectival modification indicating they are few).

As I said above, I think if anything `nmod:npmod` makes the most sense if we want "a few" as a constituent, since it would be analyzed as a full modifier NP indicating extent without a preposition. But I don't think that's really right, and honestly this just makes me want to stick to amod and leaving "a" as det to the noun in one of those "UD just likes fountainy graphs" moments (there are quite a few of these murky fountains, like "not only" (not doesn't modify only in EWT), or "to" attaching to a predicate noun/adj in things like "to be good". It's not 100% right, but it makes life a bit simpler and I've just come to accept it in the name of lexicocentrism I guess.

**Comment by nschneid** [comment]
> `det` on an adjective with no noun is the normal way of handling 'promotion' (or more traditionally, nominalization of the adjective), so I don't find it bad. It's the same as "the poor", or "the following". My concern was having something with deprel `amod` having the `det`, which this would introduce in this construction.
> 
> I think I feel pretty strongly against `fixed` for a number of reasons:
> 
>   * Whatever we think "scant" is, the construction is modifiable, so you'd be introducing potentially quite a few "fixed" expressions to an already long list

Here's CGEL (p. 392):

![image](https://user-images.githubusercontent.com/985263/117721361-c6574a00-b1ad-11eb-9447-596de00896b3.png)

Really "a few" and "a little" are special constructions that permit very limited internal modification. Maybe UD needs an `almost-fixed` relation. ;)

>   * If fixed is really only used for things where headedness is meaningless, I don't think this is such a case: I'm pretty sure "few" outranks "a"

I don't have that intuition. I think they act in concert.

>   * It obscures the fact that "a" is actually fulfilling a fairly normal role here, and erases the parallelism with "few" without an article. Currently we have "few dogs" and "a few dogs" being structurally very similar (both have an adjectival modification indicating they are few).

But if you think the indefinite article is acting as usual then it's very odd that it's modifying a plural noun, yes? You can't say "a several dogs" or "a many dogs". "A few" is simply a syntactically special expression.

**Comment by amir-zeldes** [comment]
I'd be against `flat` for the same reasons as `fixed`. It's not really similar to any of the current uses of `flat` in English, and I really don't see a reason not to treat "a" as a determiner here. "a" generally doesn't have dependents and typically modifies both nouns and nominalized adjectives, so it fits perfectly as `det`. If I wanted "a few" to be a constituent (i.e. dependency chain), I think it would be most parallel to the extent modifiers found with spatio-temporals and other quantities. Compare a possible:

![image](https://user-images.githubusercontent.com/2188635/118129291-fda24280-b3c9-11eb-90d2-08836adcb1d0.png)

To existing analyses like:

![image](https://user-images.githubusercontent.com/2188635/118129327-0abf3180-b3ca-11eb-827f-bc40261e2239.png)

![image](https://user-images.githubusercontent.com/2188635/118129375-14e13000-b3ca-11eb-910a-4e4331e6edcc.png)

I'm still not sure it's a huge improvement over what we have now, but the idea of "a" heading a `flat` expression seems wrong when it's basically just a determiner and looks exactly the same as in the independent "a few" functioning as an argument (I assume you wouldn't make that be `flat` either way, right?)

**Comment by nschneid** [comment]
So I guess the crux of the question is whether it is a nominalized adjective like "the poor" or "the British". I don't see it that way—yes you could say "the many as opposed to the few" just like "the rich as opposed to the poor", but this strikes me as an entirely different usage from "a few" as in "I bought a few". And note that while you can say "the many as opposed to the few" you cannot say "a many books", so there is something special about "a few" and "a great many".

Can the normal nominalized adjective construction be used with an indefinite article? Here are results with indefinite article + ADJ other than "few" or "little", and they look like annotation errors (attachment errors or lexemes with distinct nominal senses that I'd tag as NOUN, e.g. "a contemporary" = 'a person who is around at the same time as someone'), not a productive construction where an adjective is coerced to a nominal.
- GUM: http://match.grew.fr/?corpus=UD_English-GUM@dev&custom=609d2774623a3
- EWT: http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=609d281aead2e

**Comment by nschneid** [comment]
> I'm still not sure it's a huge improvement over what we have now, but the idea of "a" heading a `flat` expression seems wrong when it's basically just a determiner and looks exactly the same as in the independent "a few" functioning as an argument (I assume you wouldn't make that be `flat` either way, right?)

"I bought a few"—I'd want that to look similar to "I bought several". This could be achieved with `flat` + an ExtPos feature. Remember that `flat` is really asserting that there is no head syntactically—"a" and "few" are on equal footing—there's only a head in the tree for data structure convenience.

**Comment by amir-zeldes** [comment]
> I'd want that to look similar to "I bought many"

Yes, but I'd like "I bought a few" to look similar to "I bought few"...

**Comment by nschneid** [comment]
Yeah that would be achieved too with `flat` and ExtPos=ADJ.

**Comment by amir-zeldes** [comment]
I'm not sure I understood you- are you saying "a few" by itself should also be flat? If so, what about "the few" in "the few I know"? And what would you do about things like:

  * Another few weeks
  * It's been a long few days

I think keeping "a" as det to something makes the most sense given the flexibility and potential internal modifications of few.

**Comment by nschneid** [comment]
Oooh fun. "A long few days"—hadn't thought about this. I think it's yet another construction ("a" + adjective quality modifier(s) + quantifier): "a tough several days", "an unprecedented 8 days". So I would say that in that case "few" is acting as a normal quantifier-adjective, and this construction licenses a special use of "a" but I'm not sure whether to say the quantity adjective itself licenses "a". I.e. it seems like a semiproductive construction where "a" is special but not part of a frozen expression, so UD won't have the capacity to capture it and `det(days, a)` is probably the best we can do.

Note that "a long few days" and "a few long days" parse differently for me, even if they work out to meaning similar things in practice. (cf. _a long many days_ vs. _*a many long days_)

Another few weeks, the next few weeks, etc.—I think here "few" is just a regular quantifier adjective. (cf. _another several weeks_, _the next several weeks_)

**Comment by nschneid** [comment]
> I'm not sure I understood you- are you saying "a few" by itself should also be flat? If so, what about "the few" in "the few I know"?

"The few I know" the definite-article + adjective-coercion-to-nominal construction, so I think that would be `det` just like "the rich".

**Comment by amir-zeldes** [comment]
I think "a long few days" and "a few days" is the same "few".  I would consider all of the above constructions to contain the article with its normal label of det, and it think it would be the most intuitive and reliable for annotators as well.

**Comment by sylvainkahane** [comment]
If "few" works as an adjective in "my very few books", it appears that "a few" works as a determiner in "a few books" because it  blocks up the position and prevents any other determiner to appear. So I think that "a few" must be `det` with ExtPos=DET.

About the internal structure of "a few" you showed that some adjectives are possible so it is possible not to treat "a few" as a fixed expression and to analyse "a" as a `det` of "few". I also seems that "few" in "a few" acts as a NOUN in this case.  Maybe I am influenced by French where we have the ADV/NOUN "peu" which acts now as an adverb ("il lit très peu" 'he reads very little') but was a noun and appears in many fixed expressions with a determiner (_il lit un peu_ 'he read a few'; _le peu que j'en sais_ 'the little I know', etc.).

I agree with @nschneid that "a long few days" might be a different construction, where "few days" is a unit and not "a long few".

**Comment by nschneid** [comment]
> If "few" works as an adjective in "my very few books", it appears that "a few" works as a determiner in "a few books" because it blocks up the position and prevents any other determiner to appear. So I think that "a few" must be `det` with ExtPos=DET.

Oh good point. So "I bought a few books" has ExtPos=DET. What about "I bought a few"—ExtPos=ADJ or DET? Note that "many" is always tagged as ADJ, whereas "some" is always tagged as DET. Both can appear without a head noun: I bought many/some. I guess treating "a few" as DET across the board, like "some", would make sense (the many books / *the a few books / *the some books).

Re: "few" acting as a noun...I agree it can do this in some contexts (e.g. with a definite article), but in "a few" it is hard for me to say it is more noun-like than "some" or "many" apart from "a" looking superficially like an article. And a DET MWE with an internal DET feels a bit awkward to me. Seems like the more neutral solution is to say, this is a weird expression that doesn't generalize to words other than "a" + "few/little" + occasionally an internal modifier, and `flat` is a way to achieve that.

And from a native speaker intuition perspective, I am having trouble conceptualizing "few" and "little" as nouns (as opposed to "lot" or "bit").

**Comment by amir-zeldes** [comment]
Let me recap my arguments against flat/fixed:

  * It should not be fixed, because: 
    * there are many variants (a scant few, a long few, ...)
    * More generally, even if we could list a bunch of these, fixed should be used sparingly since it creates long shopping lists that annotators need to memorize
  * It is not flat, because: 
    * there is an internal modification structure for the cases with adjectives
    * if we want it to have the same analysis as cases with other determiners and cases with zero determiner, "few" has to be the head of "the"

I agree that there are interesting and subtle differences between the various constructions, but I think an average user would expect these to look similar:

I know a few
a few that I know
the few that I know
few that I know (would say that)
our last few remaining problems

Across these related constructions, "few" can be combined with most normal determiner options, suggesting that the "super-schema" of what they have in common basically calls for one of the standard English determiners (incl. zero). I think that making "few" be a child of the determiner in only some of this is asking for annotator disagreements and parsing errors, and I don't see any real benefit. 

If the goal is to have "a few" as a phrase, then I think it should follow the normal determiner as child + det option, which would maintain the status quo for independent "a few" and keep parallelism to the other constructions with "few".

**Comment by nschneid** [comment]
Let me lay out my argument for the different constructions before getting to whether it is practical or not for annotators.

I think the key syntactic tests for MWE status are:
- Can the det+adj combination be substituted with "some"?
- Can the adj be substituted freely with other quantity adjectives (several, many, etc.)?

I would advocate the MWE analysis (with flat) only for cases that pass the first test and fail the second, showing that the expression is particular to "a" + "few"/"little" and that together they function like "some". For example:

1. I bought a few books.
   * MWE: I bought some books; *I bought a several books
   * `obj(bought, books)`, `flat(a/DET, few/ADJ)`, a+few: ExtPos=DET, `det(books, a)`
2. I bought a few. 
   * MWE: I bought some; *I bought a several
   * `obj(bought, few)`, `flat(a/DET, few/ADJ)`, a+few: ExtPos=DET
3. I bought few/many/several books. — not MWE: `obj(bought, books)`, `amod(books, few/ADJ)`
4. I bought the few/many/several books (that I wanted). — not MWE: `obj(bought, books)`, `amod(books, few/ADJ)`, `det(books, the)`
5. I stayed a long few/many/several days. — not MWE: `obj(stayed, days)`, `det(days, a)`, `amod(days, long)`, `amod(days, few)`

So while on the surface these are similar:

> a few that I know
> the few that I know

the above tests distinguish them (cases 2 & 4).

Now, we have the question of whether it is practical to have UD annotators remember to use an anomalous deprel of "a" in the first two cases (flat rather than det). I can see the point that it requires a lot of nuance. While I think flat is more of a "proper" MWE treatment, I think it is OK to fudge a bit on the internal structure deprels to make them look more like the typical uses of those deprels, so I could envision the following compromise that does away with `flat`:

1. I bought a few books. `obj(bought, books)`, __`det(few/ADJ, a/DET)`__, few: ExtPos=DET, `det(books, few)` [whole expression considered a determiner to address @sylvainkahane's point]
2. I bought a few. `obj(bought, few)`, __`det(few/ADJ, a/DET)`__, few: ExtPos=DET
3. I bought few/many/several books. `obj(bought, books)`, `amod(books, few/ADJ)`
4. I bought the few/many/several books (that I wanted). — not MWE: `obj(bought, books)`, `amod(books, few/ADJ)`, `det(books, the)`
5. I stayed a long few/many/several days. — not MWE: `obj(stayed, days)`, `det(days, a)`, `amod(days, long)`, `amod(days, few)`

Basically the principle would be, "a" usually modifies "few" rather than the head noun. We already do this if there is no head noun, so the change would make things more consistent.

**Comment by sylvainkahane** [comment]
I understand the reluctance of @amir-zeldes to use `fixed` for "a few" because the internal structure is quite clear and the expression is not completely frozen, seing that some ADJ can modify "few". But according to UD choices, I think that "a few" must be considered as a fixed expression. The main argument is the fact that "few" and "a few" don't have exactly the same distribution: "few" works as an `amod`, while "a few" works as a `det`.

(Note that in SUD we decided not to use the relation `fixed`, and to indicate the internal syntactic structure of idioms and use features `PhraseType=Idiom` and `ExtPos` on the head of the idiom and `InIdiom=Yes` for the the words inside the idiom. It means that we will use the relation `det(few, a)` and add `PhraseType=Idiom` and `ExtPos=DET` on "few".)

@nschneid I don't think `flat` is a possible solution. If we consider "a few" as a fixed expression, the relation must be `fixed`, if not, the relation must be `det(few, a)`.

**Comment by amir-zeldes** [comment]
I agree with @sylvainkahane that there is internal structure, but I think that rules out not only `flat`, but also `fixed`, since UD stipulates that `fixed` goes left to right from the first token. If it is not completely frozen, as you say above, then it is also not fixed. The fact that "few" and "a few" are not distributed the same is true of many items with and without an article, but still, when the article is there I think it is the regular determiner, and the NP "a few" has a certain function then, which can be non-identical to the function of "few".

The about @nschneid 's suggestion: I would feel uncomfortable doing a chain of `det`s, which is otherwise unattested in English. I understand the appeal of treating it like a complex determiner, but I think here we run up against UD's token-centric conventions, which also mandate that "hours" in "dance two hours" is not `advmod`, since "hours" in itself is not an adverb. I think since we are dealing with a phrasal modifier of nouns, it should be `nmod:npmod`, and this also avoids the awkward `det` chain. It's also not unusual for an nmod subtype to take up a specifier position, just like possessive `nmod:poss` can mark a genitive possessive or a possessive article (my, your etc.)

Finally, specifically for 5 above, if we want a phrasal "a few" analysis, then I don't think "a" should modify "days" (though currently it would). I think although semantically it is the days that are long, syntactically we have:

det(few, a)
amod(few, long)
nmod:npmod(days,few)

**Comment by nschneid** [comment]
> The about @nschneid 's suggestion: I would feel uncomfortable doing a chain of `det`s, which is otherwise unattested in English. I understand the appeal of treating it like a complex determiner, but I think here we run up against UD's token-centric conventions, which also mandate that "hours" in "dance two hours" is not `advmod`, since "hours" in itself is not an adverb. I think since we are dealing with a phrasal modifier of nouns, it should be `nmod:npmod`, and this also avoids the awkward `det` chain. It's also not unusual for an nmod subtype to take up a specifier position, just like possessive `nmod:poss` can mark a genitive possessive or a possessive article (my, your etc.)

I see your point but `nmod:npmod` indicates we are analyzing "a few" as a nominal expression, which seems wrong when it's in specifier position ("a few books"). To me it is bare "a few" ("I bought a few") that is coerced from a compound determiner into a nominal, not that it is always a nominal.


> Finally, specifically for 5 above, if we want a phrasal "a few" analysis, then I don't think "a" should modify "days" (though currently it would). I think although semantically it is the days that are long, syntactically we have:
> 
> det(few, a)
> amod(few, long)
> nmod:npmod(days,few)

But this makes "a long few" into a constituent, which I don't see any evidence for.

- The event is over how many days, several or just a few?
- *The event is over how many days, several or just a long few?

**Comment by amir-zeldes** [comment]
I think it is a nominal expression, that's why it can take "a" in the first place, no? Compound determiners generally still have an underlying part of speech, and I think it's the same "a few" as always (so either a noun, or an adjective nominalized into a noun, but either way a nominal).

> But this makes "a long few" into a constituent, which I don't see any evidence for.

Some examples with an adjective but without a noun:

  * Gartner , 53 , had planned to share the news of her surgery and treatments with **a select few** , but her appearance changed that .
  * If all of this seems like the concern of **a specialized few** , consider that plant breeders shape nearly every food we eat
  * How can I explain to him that because of **a bad few** , from both sides of the fence , we will not be able to walk freely in my home town ?
  * I believe Tesco have **a fair few** .
  * With the exception of **a rare few** , publishing is a long haul game for everyone .

I feel really put off by the look of det <- det, especially with it being unparalleled elsewhere in English dependency analyses, and probably being typologically very rare across UD. In fact, I'm starting to think that if it is so unclear then maybe the best solution is actually the current one, which is possibly a bit naive but at least easy to explain and consistent across constructions.

**Comment by nschneid** [comment]
"a select/specialized/fair/rare few"—these are all internal modification of "a few" (which is why it's arguably not `fixed`). For me the modifiers allowed here are limited to explaining *how* few or particularized the amount is, not arbitrary properties of the thing being quantified.

The "bad few" example is ungrammatical for me. Perhaps a different dialect.

**Comment by amir-zeldes** [comment]
> the modifiers allowed here are limited to explaining how few or particularized the amount is, not arbitrary properties 

I think those are semantic distinctions and not syntactic ones. Once the NN is dropped, the only thing left for modification is "few" either way. I'm also not really sure that's true of some of these, for example being a "select" few is a property of the thing counted, not of the count itself. But if you want clearer examples of semantically 'non-quantitative' modifiers, like "bad", those aren't too hard to find:

  * Does the Genesis concept of one-flesh union capture a real human good achievable only by a man and a woman , or is it gerrymandered to exclude the loves of a hapless few ?
  * Magnus becomes their protector . And for a dangerous few , she is their captor .
  *  The fact that Nazneen , who has an arranged marriage to a much older man , embarks on an affair with a much younger man was considered a negative portrayal by a predictable few .
  * The violence and vandalism came from a militant few , believes one student .
  * Remember that this is just a starting point for some ; all that 's needed for a lucky few .
  * This must have been done by a mindless few .

**Comment by nschneid** [comment]
Interesting. This is a sense of "few" that at least some dictionaries categorize as a noun. I think it prototypically means a minority of *people*. And I notice the adjective is often evaluative; maybe this is part of the construction's core meaning (singling a small group of people out as aberrant).

Could it refer to a small quantity in general? I'm not sure:

?Though I enjoy eating berries, occasionally the experience is marred by a sour few.

??You can have all the sweet berries; I'll just have a sour few. (Better: a few sour ones)

??While the treatment used to require upwards of 12 visits, now it can be completed in only a short few. 

Anyway I'm becoming convinced that there's a continuum of grammaticalization at work here, with several interrelated and nuanced constructions, which is why categorization is tricky.

**Comment by amir-zeldes** [comment]
Yes, the more I look into examples the more I realize how productive this construction is, and it's hard to find clear boundaries between classes. For me this all speaks for just leaving it alone, and definitely staying away from fixed solutions. 

BTW there are plenty of examples of non-humans with adjectives, for example:

  * Organizations: While not quite dinosaurs , the literally thousands of big bands that dotted the American landscape from the 1920 's through the 50 's have dwindled to a precious few
  * Buildings: She likes churched so we went to a fair few , and this was indeed one of the most beautiful .
  * Animals: I 'm not sure how many green budgies have had the name Freddie Halfpenny as they are not long lived creatures , but a good few I have no doubt . 
  * Cars: i can remember austin sending a fair few 6r4s through measham car auctions in the 80 s

**Comment by nschneid** [comment]
OK but all of these seem to me like quantity-modifying adjectives rather than property adjectives. "a good few (budgies)" doesn't mean a few budgies that are good.

I'm polling people on social media to see how they feel about the grammaticality of some of these. And getting mixed responses. Clearly it's complicated!

**Comment by amir-zeldes** [comment]
Hm, I see, well if you want 1. the few construction, 2. for the noun to be missing, 3. for there to be an adjective modifying few, 4. for that adjective to be semantically non-quantitative and 5. for the entire phrase to stand for a non-human... that's just going to be very rare by virtue of the chain rule - but I'm not sure if that's important for the syntax. 

If it is, then wading through corpus examples I can offer these cases, some are more borderline than others:

  * Works of art (by human artists, could be metonymy?): The contributions from artists living in Camden and Islington are certainly significant in number , perhaps as always , but not necessarily totally memorable . To name **a worthy few** , take a look at the work of Tom Fairs , Maiko Hatano , Robina Barson , Eileen Hogan , and Norman Miller .
  * Corporations (probably anthropomorphized): With our elections now in the hands of **a corporate few** , all that has changed and is only getting worse .
  * Abstract:  doing your homework reduces the number of surprises to **a manageable few** .
  * Abstract or concrete object (not sure if it's the vanity or its style): You can opt to go with a simple designed vanity like mentioned above , or you can go with a more stylized vanity that is themed like Modern , Contemporary , Antique to name just **a popular few** .

But TBH your example "a good few bad budgies" suggests to me that adjectives belonging syntactically to the 'lexical' noun should appear between it and "few", so if "long" were a syntactic modifier of "days", we should get "a few long days", not "a long few days". Semantics aside, I think the difference in position probably corresponds to a syntactic subordination difference. 

That all being said, I'm fine with keeping everything always being a dependent of the lexical noun if available, and of "few" if not, which is the status quo.

**Comment by nschneid** [comment]
Yeah I'm hearing various levels of discomfort about some of these...a lot of "I definitely wouldn't say that but maybe someone would". So there's probably no fixed set of boundaries that all speakers would agree on, just some more prototypical and less prototypical cases.

Even if I had a clear linguistic analysis of the full range of constructions, it probably wouldn't be obvious how to map them to UD in practice. So let's just stick with the status quo for now.
